tutorials/xr/openxr_passthrough.rst

```diff
@@ -15,12 +15,12 @@ Passthrough extension
 ---------------------
 
 OpenXR has a vendor extension for passthrough submitted by Meta.
-Currently this extension is only supported on Quest and PICO but may be adopted by other headsets in the future.
+Currently, this extension is only supported on Quest and PICO but may be adopted by other headsets in the future.
 
 :ref:`XRInterface <class_xrinterface>` has entry points for passthrough so different interfaces can implement this feature.
-For :ref:`OpenXRInterface <class_openxrinterface>` the meta passthrough extension is implemented here.
+For :ref:`OpenXRInterface <class_openxrinterface>` is the Meta passthrough extension previously mentioned.
```

Comment:
This is wrong, the `OpenXRInterface` class isn't a passthrough extension, also this sentence isn't correct grammatically, it now says "because ... Is the", as in a poetic term, as opposed to "for this case"